src/Resources/config/routing.yml

```diff
@@ -1,8 +1,14 @@
 bitbag_sylius_wishlist_plugin_shop_wishlist_list_products:
     path: /wishlist
+    methods: [GET]
     defaults:
         _controller: bitbag_sylius_wishlist_plugin.controller.action.list_wishlist_products
 
+bitbag_sylius_wishlist_plugin_shop_wishlist_add_products:
+    path: /wishlist/add/products/all
```

Comment:
Control behavior using HTTP verbs: `POST /wishlist/products` means the same as `/wishlist/add/products…`.

Reply:
There is already POST  /wishlist/add/{productId}.